bioio_bioformats/tests/test_reader.py

```diff
@@ -196,7 +187,7 @@
             (1, 3, 1, 1024, 1024),
             np.uint16,
             dimensions.DEFAULT_DIMENSION_ORDER,
-            ["Red", "Green", "Blue"],
+            ["Channel:0:0", "Channel:0:1", "Channel:0:2"],
```

Comment:
This is actually a return to what we had before. The last commit to main updated it to ["Red", "Green", "Blue"]